### ISSUE TITLE
old-style engine.cfg choking on 'img' identifier

### ISSUE THREAD

**Issue by adolson** [godotengine/godot#3798]
**Operating system or device:**
Linux 64-bit

**Issue description** (what happened, and what was expected):

Old project won't show in Project Manager now. Get this error:

```
ERROR: load: ConfgFile::load - /home/dana/Projects/HiddanOjbexxRetchro/engine.cfg:6 error: Unexpected identifier: 'img'.
   At: core/io/config_file.cpp:180.

```

Expected it would import into the Project Manager list.

**Steps to reproduce:**

Try to import project with custom boot logo in engine.cfg

**Link to minimal example project** (optional but very welcome):

https://gitlab.com/shine-upon-thee/hiddan-ojbexx-retchro/repository/archive.zip?ref=master

**Comment by reduz** [comment]
custom boot logo in engine.cfg is no longer supported, but I'm surprised this no longer works, I added compatibility..

**Comment by reduz** [comment]
Will check later, but it's weird because I added the code to parse it..

**Comment by adolson** [comment]
Yeah, I figure I could probably just load my project by simply deleting that line. But I looked at the code and saw it was supposed to be supported, so that's why I reported it, as I'm sure it's something good to fix before 2.0 release.

**Comment by adolson** [comment]
OK, so, I don't know WTF happened yesterday, but this seems to be fine with a fresh checkout of the 2.0 stable branch. Closing.
